src/pages/MenuPage.js

```diff
@@ -15,7 +15,47 @@ import { SETTINGS_KEYS } from '../settings';
 
 import globalStyles, { APP_FONT_FAMILY, SHADOW_BORDER, GREY, WARMER_GREY } from '../globalStyles';
 
-const { SYNC_SITE_NAME } = SETTINGS_KEYS;
+/* eslint-disable global-require */
+
+const { SYNC_SITE_NAME, THIS_STORE_CUSTOM_DATA } = SETTINGS_KEYS;
```

Comment:
I'm not sure about holding store custom data. I can see why we are storing this data in `custom_data` in 4D. I don't see why bringing this into mobile is a good idea. 

My ideal:
`THIS_STORE_MODULES: ['VACCINES', 'DISPENSARY']`

Although I can see there could *potentially* be a use-case for having additional information about a certain stores modules.

`THIS_STORE_MODULES: { VACCINE: { }, DISPENSARY: { } }`

--

Keep with the current way: I think:
`THIS_STORE_CUSTOM_DATA: { modules: ['VACCINE'] }`
or 
`THIS_STORE_CUSTOM_DATA: { vaccine: { enabled : true }}`

The current shape/names seem a little odd

Reply:
good point, i'll make an issue

Reply:
https://github.com/openmsupply/mobile/issues/880